src/HotChocolate/ApolloFederation/src/ApolloFederation/Extensions/ApolloFederationFieldDescriptorExtensions.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using HotChocolate.ApolloFederation;
+using HotChocolate.Language;
+using HotChocolate.Types;
+using HotChocolate.Types.Descriptors;
+using HotChocolate.Types.Descriptors.Definitions;
+using DirectiveLocation = HotChocolate.Types.DirectiveLocation;
+
+namespace HotChocolate.Types
```

Comment:
@fredericbirke I have moved this to HotChocolate.Types to follow our convention. 

HotChocolate.Types => Descriptor Extensions
HotChocolate => SchemaBuilder Extensions
Microsoft.Extensions.DependencyInjection => IRequestExecturorBuilder Extensions and important IServiceCollection extensions.

Reply:
Acknowledged